core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10136,6 +10136,55 @@ private void checkLiteral2(String expression, String expected) {
     sql(sql).ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-7050">[CALCITE-7050]
+   * Invalid unparse for FULL JOIN in MySQLDialect</a>. */
+  @Test void testUnsupportedFullJoin() {
+    // MySQL full join would transform to left join union right join
+    String sql = "select *\n"
```

Comment:
Select the specific columns will make this test case clearer.

Reply:
OK，Had simplied the test case to make it more clearer.